packages/jsii-pacmak/lib/targets/dotnet/dotnetgenerator.ts

```diff
@@ -740,7 +744,10 @@ export class DotNetGenerator extends Generator {
       elementType: spec.TypeReference,
       parameterName: string,
     ) {
-      const varName = `__item_${descr.replace(/[^a-z0-9_]/gi, '_')}`;
+      const varName = `__item_${createHash('sha256')
+        .update(descr)
+        .digest('hex')
+        .slice(0, 6)}`;
```

Comment:
Unrelated, but makes the generated code more readable, which makes it less likely that a bad change is missed by human scrutiny.